src/repository/file.repository.ts

```diff
@@ -34,4 +34,14 @@ export default class FileRepository {
   public async insert(fileData: FileCreationAttributes): Promise<UploadedFile> {
     return await this.storage.insertFile(fileData);
   }
+
+  /**
+   * Get file location by key and type
```

Comment:
Useless description. Describe why do we need this method, its use case, please.

Reply:
updated description